kolibri_instant_schools_plugin/assets/src/views/select-profile-page/profiles-list.vue

```diff
@@ -1,34 +1,47 @@
 <template>
 
-  <div>
-    <div
-      v-for="profile in profiles"
-      class="profile-card"
-      :class="{ 'profile-card-disabled': disabled }"
-      :key="profile.username"
-      role="button"
-      @click="selectProfile(profile)"
-    >
-      <div class="profile-icon">
-        <mat-svg
-          category="social"
-          name="person"
-          class="person-icon"
+  <table class="table">
+    <tr v-for="profile in profiles" :key="profile.username">
+      <!--
+        Adding click on this table cell as a convenience.
+        Accessibility is not affected because it's redundant with
+        the 'select' button in the next table cell.
+      -->
+      <td @click="selectProfile(profile)" class="name-wrapper">
```

Comment:
Not too sure about this. If we do want to make it clickable it should look clickable eg drop-shadow/cursor/etc